views/js/runner/plugins/security/fullScreen.js

```diff
@@ -223,8 +223,14 @@ define([
          * Initializes the plugin (called during runner's init)
          */
         init: function init() {
-            var testRunner = this.getTestRunner();
-            var waitingForUser = false;
+            const testRunner = this.getTestRunner();
+            const dialogParams = {};
+            const config = this.getConfig();
+            let waitingForUser = false;
+
+            if (config && config.focus) {
+                dialogParams.focus = config.focus;
+            }
```

Comment:
```suggestion
            const { focus } = this.getConfig();
            const dialogParams = { focus };
            let waitingForUser = false;
```

Reply:
Not sure about this, since in the case of config or focus undefined it will create a dialogParams = {focus: undefined}, this is going to override default config later on.